Dockerfile.dev

```diff
@@ -21,7 +21,8 @@ RUN /bin/bash -c "source activate sqlflow-dev && python -m pip install \
     grpcio-tools \
     tensorflow==2.0.0-alpha0 \
     mysql-connector-python \
-    pyhive \
+    impyla \
+    thrift \
```

Comment:
`pyhive` relies on `thrift`, or a `ModuleNotFoundError` will be reported, but `impyla` does not.
Since we won't use `pyhive` anymore, `thrift` should be removed.